stdlib/public/TensorFlow/Ops.swift

```diff
@@ -386,49 +388,56 @@ public extension Tensor where Scalar : Equatable {
   }
 }
 
-public extension Tensor where Scalar == Bool {
-  /// Performs a logical NOT operation element-wise.
+infix operator ≈ : ComparisonPrecedence
+
+public extension Tensor where Scalar : BinaryFloatingPoint & Equatable {
+  /// Returns a `Tensor` of Boolean values indicating whether the elements of
+  /// `self` are approximately equal to those of `other`.
   @inlinable @inline(__always)
-  static prefix func ! (x: Tensor) -> Tensor {
-    return Raw.logicalNot(x)
+  func elementsApproximatelyEqual(_ other: Tensor,
+                                  tolerance: Double = 0.00001) -> Tensor<Bool> {
+    return Raw.approximateEqual(self, other, tolerance: tolerance)
   }
 
-  /// Performs a logical AND operation element-wise.
-  /// - Note: `&&` supports broadcasting.
+  /// Returns a Boolean value indicating whether the value of the first argument
+  /// is lexicographically approximately equal to that of the second argument.
+  /// Errors no larger than `0.00001` are tolerated.
   @inlinable @inline(__always)
-  static func && (lhs: Tensor, rhs: Tensor) -> Tensor {
-    return Raw.logicalAnd(lhs, rhs)
+  static func ≈ (lhs: Tensor, rhs: Tensor) -> Bool {
+    return lhs.elementsApproximatelyEqual(rhs).all()
```

Comment:
As you know, I'm not a huge fan of innovating in operators without a carefully thought out and well motivated plan.  I'd strongly prefer that all unicode operators have a directly analogous method so people can choose to use the sugar if they want, but aren't punished too much if they don't.

@stephentyrone will point out that comparing to 0.00001 isn't principled, and can be problematic in practice.  Is this precedented from the TF world?

Reply:
TF `ApproximateEqual` has a default error tolerance which is `0.00001`.